src/utils/format.utils.ts

```diff
@@ -79,3 +79,7 @@ export const slicedBased58Key = (key: PublicKey | string) => {
 
 	return key.toBase58().slice(0, 5) + ".." + key.toBase58().slice(-5);
 };
+
+export function capitalizeFirstLetter(string) {
```

Comment:
there is a lodash function for this. this looks correct though. what is your opinion? don't depend on dependencies for even stupid stuff like this or keep the codebase small and lean and just leverage existing code without reinventing the wheel?

Reply:
Ah yeah, I'm not in the habit of using lodash. 
For me it's fine to use lodash for utils like this. Less stuff for us to test/maintain. 

What I do want then is to add it explicitly to `package.json`, I'm not comfortable with using a package that is a dependency of another package. 

So, I'll do two things then:
1. add lodash to `package.json`
2. remove this util function + corresponding test and use lodash instead